src/plugins/intel_npu/src/al/include/intel_npu/npuw_private_properties.hpp

```diff
@@ -488,6 +488,15 @@ static constexpr ov::Property<uint32_t> max_lora_rank{"NPUW_LLM_MAX_LORA_RANK"};
  */
 static constexpr ov::Property<bool> optimize_v_tensors{"NPUW_LLM_OPTIMIZE_V_TENSORS"};
 
+/**
+ * @brief
+ * FIXME: Should be removed.
```

Comment:
Could I please ask you to elaborate on this `FIXME` and add a reason why this should be removed in the comment?

Reply:
copy paste from optimise_v_tensor flag